systemSmartContractsConfig.toml

```diff
@@ -1,6 +1,6 @@
 [StakingSystemSCConfig]
-    GenesisNodePrice = "2500000000000000000000" #2.5KERD
-    MinStakeValue = "100000000000000000000" #100ERD
+    GenesisNodePrice = "2500000000000000000000" #2.5K eGLD
+    MinStakeValue = "100000000000000000000" #100 eGLD
```

Comment:
Comment is wrong

Reply:
`100000000000000000000` -> `100,000_000_000_000_000_000 eGLD`